docker/django-start.sh

```diff
@@ -27,7 +27,9 @@ django_start() {
     if [ "${DEPLOY_ENV}" == "dev" ]; then
         ./manage.py runserver 0.0.0.0:8000
     else
-        ./manage.py collectstatic -c --noinput
+        if [ "${DJANGO_COLLECT_STATIC}" == "yes" ]; then
+            ./manage.py collectstatic -c --noinput
+        fi
```

Comment:
Do you want to make this completely orthogonal to `DEPLOY_ENV` maybe, by unnesting the `if` structure? It wouldn't be common, but I could imagine doing dev testing with a real static file server...